Car-slider.html

```diff
@@ -0,0 +1,238 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <style>
+      * {
+        box-sizing: border-box;
+        margin: 0;
+        padding: 0;
+        outline: none;
+      }
+
+      .car__info__wrapper {
+        display: inline-block;
+        padding-left: 5px;
+        padding-right: 5px;
+      }
+
+      .card__image__content {
+        background-color: rgb(248, 246, 246);
+        border-radius: 5px;
+        border: 1px solid rgb(202, 202, 202);
+      }
+
+      .card__container {
+        width: 63%;
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+        border-radius: 5px;
+      }
+
+      .car__information {
+        padding: 16px 12px 20px;
+      }
+
+      .car__name {
+        font-size: 16px;
+        color: #484848;
+        font-weight: 400;
+      }
+
+      .car__price {
+        font-size: 16px;
+        margin-top: 8px;
+        font-weight: 700;
+        color: #484848;
+      }
+
+      .car__avg__price {
+        font-size: 13px;
+        margin-top: 8px;
+        color: #6f6f6f;
+      }
+
+      .car__city__price {
+        margin-top: 8px;
+      }
+
+      .citywise__price {
+        color: #0288d1;
+        text-decoration: none;
+      }
+
+      .onwards__text {
+        font-size: 13px;
+        font-weight: 400;
+        color: #aaa;
+        padding-top: 2px;
+        padding-left: 3px;
+      }
+
+      .previous__card__slider,
+      .next__card__slider {
+        height: 32px;
+        width: 32px;
+        background-color: #d7d7d7;
+        position: absolute;
+        z-index: 1;
+        top: 50%;
```

Comment:
This won't perfectly center it from top

Reply:
if I used left and translate the slider will show up of center of middle card that's why I haven't used it we can change the position separately to both side it will make them center

Reply:
you can specifically mention translateX(-50%) which will only translate in Y axis